test/guardian/plug/ensure_permissions_test.exs

```diff
@@ -12,59 +13,54 @@ defmodule Guardian.Plug.EnsurePermissionTest do
     end
   end
 
-  @expected_failure TestHandler
-  @failure [handler: @expected_failure]
-
-  test "does not call the on failure when the permissions are present" do
-    opts = EnsurePermissions.init(@failure ++ [ default: [:read, :write] ])
+  setup do
+    conn = conn(:get, "/foo")
+    {:ok, %{conn: conn}}
+  end
 
+  test "doesnt call unauthorized when permissions are present", %{conn: conn} do
     pems = Guardian.Permissions.to_value([:read, :write])
     claims = %{ "pem" => %{ "default" => pems } }
 
-    expected_conn = :get
-                    |> conn("/get")
-                    |> Guardian.Plug.set_claims({ :ok, claims })
-                    |> Plug.Conn.fetch_query_params
-                    |> EnsurePermissions.call(opts)
+    expected_conn =
+      conn
+      |> Guardian.Plug.set_claims({ :ok, claims })
```

Comment:
@aaronrenner no, I don't think that's necessary.  We can update them as we go through and change code.